web/massastation/src/pages/Store/StationSection/StationPlugin.tsx

```diff
@@ -97,32 +103,73 @@ export function StationPlugin({
     ],
   };
 
-  const argsOff = {
-    preIcon: <img src={logoURL} alt="plugin-logo" />,
-    topAction: (
-      // we use customClass because "disabled" doesn't let us click on the button to turn it back on
-      <Button
-        onClick={(e) => updatePluginState(e, PLUGIN_START)}
-        customClass="bg-primary text-tertiary"
-        variant="toggle"
-      >
-        off
-      </Button>
-    ),
-    title: name,
-    subtitle: author,
-    subtitleIcon: massalabsNomination.includes(author) ? <Certificate /> : null,
-    version: `v${version}`,
-    content: [
-      <Button variant="icon" disabled>
-        <FiArrowUpRight />
-      </Button>,
-      <Button variant="icon" onClick={() => deletePlugin({})}>
-        <FiTrash2 />
-      </Button>,
-    ],
-  };
-  return <Plugin {...(status === PluginStatus.Up ? argsOn : argsOff)} />;
+  switch (status) {
+    case PluginStatus.Down:
+      args = {
+        preIcon: <img src={logoURL} alt="plugin-logo" />,
+        topAction: (
+          // we use customClass because "disabled" doesn't let us click on the button to turn it back on
+          <Button
+            onClick={(e) => updatePluginState(e, PLUGIN_START)}
+            customClass="bg-primary text-tertiary"
+            variant="toggle"
+          >
+            off
+          </Button>
+        ),
+        title: name,
+        subtitle: author,
+        subtitleIcon: massalabsNomination.includes(author) ? (
+          <Certificate />
+        ) : null,
+        version: `v${version}`,
+        content: [
+          <Button variant="icon" disabled>
+            <FiArrowUpRight />
+          </Button>,
+          <Button variant="icon" onClick={() => deletePlugin({})}>
+            <FiTrash2 />
+          </Button>,
+        ],
+      };
+      break;
+    case PluginStatus.Crashed:
+      args = {
+        preIcon: <img src={logoURL} alt="plugin-logo" />,
+        topAction: (
+          <Button
+            disabled
+            onClick={(e) => updatePluginState(e, PLUGIN_START)}
+            customClass="bg-primary text-tertiary"
+            variant="toggle"
+          >
+            off
+          </Button>
+        ),
+        title: name,
+        subtitle: author,
+        subtitleIcon: massalabsNomination.includes(author) ? (
+          <Certificate />
+        ) : null,
+        version: `v${version}`,
+        content: [
+          <Tooltip
+            className="mas-tooltip"
+            content="The plugin is not working, please uninstall it and install it again."
+            icon={<FiAlertCircle className="text-s-error" />}
+          />,
```

Comment:
@pivilartisant this is not what design requires, could you have a look? I think we need to modify the uikit

Reply:
<img width="662" alt="Screenshot 2023-09-21 at 11 14 50" src="https://github.com/massalabs/station/assets/90157528/2c158fa5-b9ac-4541-b6f6-cd150847f10b">

It's looking good to me, what are the modifications you would like to do ?

Reply:
text has been modified